tests/base_scraper/test_base_scraper.py

```diff
@@ -67,25 +67,23 @@ def test_not_implemented(tmpdir, attr):
         scraper.build_filename('invalid binary')
 
 
-@pytest.mark.skip(reason="Permanent failure due to mozqa.com not available anymore (#492)")
-def test_invalid_authentication(tmpdir):
-    basic_auth_url = 'http://mozqa.com/data/mozqa.com/http_auth/basic/'
-    scraper = mozdownload.DirectScraper(destination=str(tmpdir), url=basic_auth_url)
+def test_invalid_authentication(httpd, tmpdir):
+    test_url = urljoin(httpd.get_url(), 'http_auth')
+    scraper = mozdownload.DirectScraper(destination=str(tmpdir), url=test_url)
     with pytest.raises(requests.exceptions.HTTPError):
         scraper.download()
 
 
-@pytest.mark.skip(reason="Permanent failure due to mozqa.com not available anymore (#492)")
-def test_valid_authentication(tmpdir):
+def test_valid_authentication(httpd, tmpdir):
     username = 'mozilla'
     password = 'mozilla'
-    basic_auth_url = 'http://mozqa.com/data/mozqa.com/http_auth/basic/'
+    test_url = urljoin(httpd.get_url(), 'http_auth?username={0}&password={1}'.format(username, password))
```

Comment:
Adding the username and password here already authenticates the user. It should be the plain URL without any parameters.